daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -141,7 +146,9 @@ type Status struct {
 	// UdevSyncSupported is true if sync is supported.
 	UdevSyncSupported bool
 	// DeferredRemoveEnabled is true then the device is not unmounted.
-	DeferredRemoveEnabled bool
+	DeferredRemoveEnabled      bool
```

Comment:
Ok, I will put a comment to highlight difference between Deferred Remove and Deferred Delete operation.